### ISSUE TITLE
Missing package.json

### ISSUE THREAD

**Issue by peterruijter** [lagoon-road/lr-examples#1]
Hey Roy, 
I'm missing the package.json in this repro. 
So it will be hard to follow the instructions on https://lagoonroad.com/guide/setup with this repro. Off course I can guess what should be in the package.json, but to keep the guide nice and easy this would help a lot. ;-)

Cheers!

Peter

**Comment by peterruijter** [comment]
Better read the docs will help most of the time ;-)

**Comment by braadworst** [comment]
@Peter, got it running?

**Comment by peterruijter** [comment]
Yep with Node v 7

**Comment by peterruijter** [comment]
In v 6 it complains about async and await, without babel activated
